reference/tls/cleartext-redirection.md

```diff
@@ -21,7 +21,38 @@ Ambassador Edge Stack exposes configuration for this in two ways:
 
 Typically, port-based redirection is the preferred method since it is simpler to manage and will work with all use cases. Redirecting based off of the `x-forwarded-proto` header requires an L7 load-balancer or proxy in front of Ambassador Edge Stack to set that header.
 
-## Port-based Redirection
+## Using the `Host` resource
+
+As of Ambassador API Gateway and Edge Stack 1.0.0, the [`Host`](/reference/host-crd) resource is used to configure how Ambassador handles cleartext on a domain. You can now configure Ambassador to deny, allow, or redirect cleartext to HTTPS on a domain using a single resource.
+
+The following `Host` gives an example of how to enable Ambassador to redirect cleartext to HTTPS on the `host.example.com` domain. 
+
+```yaml
+---
+apiVersion: getambassador.io/v2
+kind: Host
+metadata:
+  name: example-host
+spec:
+  hostname: host.example.com
+  acmeProvider:
+    email: julian@example.com
+  insecure:
+    action: Redirect
+```
+
+**Note:** The default behavior when Ambassador is terminating TLS is to redirect cleartext for all `Host`s
```

Comment:
The default behavior during TLS termination is to redirect cleartext for all `Host`s.

Reply:
I do not know if I agree with the suggested edit but it did make me aware of how the point I was trying to make was being missed. 

Ambassador will actually default to redirecting cleartext whether it is actively handling tls termination or the load balancer is. 

You must override this default any time you want ambassador to do something different with cleartext traffic.

Please review the new change 🙂